### PR TITLE
Changing UserMissions Model

### PR DESCRIPTION
**Pull request by jedtan**
This nests the tabStreak object in the model.

### CODE REVIEW

File: graphql/database/missions/UserMissionModel.js [hunk 1]
```diff
@@ -53,21 +55,33 @@ class UserMission extends BaseModel {
...
-        .description(`count of longest tab streak in current mission`),
-      currentTabStreak: types
-        .number()
-        .integer()
-        .default(self.fieldDefaults.currentTabStreak)
-        .description(`count of the current tab streak in current mission`),
+      tabStreak: types
+        .object({
+          longestTabStreak: types.number().integer(),
+          currentTabStreak: types.number().integer(),
```
Comment (kmjennison) [inline graphql/database/missions/UserMissionModel.js:61]: @jedtan To know if a tab is part of the current tab streak, would we check that `User.lastTabTimestamp` is <24 hours ago?

**Review by amaliwanag2 — APPROVED** [review]

File: graphql/database/missions/UserMissionModel.js (hunk 1 above)
Comment (jedtan) [reply graphql/database/missions/UserMissionModel.js:61]: Yep, or the recent day object on Max Tabs

Comment (kmjennison) [reply graphql/database/missions/UserMissionModel.js:61]: @jedtan Awesome, LGTM!

Comment (jedtan) [reply graphql/database/missions/UserMissionModel.js:61]: (within the same object so you'll have the ground truth for streaks on one single model.

**Pull request merged**